bundles/org.openhab.binding.homewizard/src/main/resources/OH-INF/i18n/homewizard.properties

```diff
@@ -5,25 +5,48 @@ addon.homewizard.description = This binding provides access to the data provided
+thing-type.homewizard.p1_wifi_meter.channel.active_current_l2.label = Current L2
+thing-type.homewizard.p1_wifi_meter.channel.active_current_l3.label = Current L3
+thing-type.homewizard.p1_wifi_meter.channel.active_voltage.label = Active Voltage
+thing-type.homewizard.p1_wifi_meter.channel.active_voltage_l1.label = Active Voltage L1
+thing-type.homewizard.p1_wifi_meter.channel.active_voltage_l2.label = Active Voltage L2
+thing-type.homewizard.p1_wifi_meter.channel.active_voltage_l3.label = Active Voltage L3
+thing-type.homewizard.p1_wifi_meter.channel.long_power_failures.label = Long Power Failures
+thing-type.homewizard.p1_wifi_meter.channel.long_power_failures.description = This channel provides the count of long power failures.
+thing-type.homewizard.watermeter.label = HomeWizard Wi-Fi Watermeter
+thing-type.homewizard.watermeter.description = This thing provides the measurement data that is available through the http interface of a HomeWizard Watermeter.
 thing-type.homewizard.watermeter.label = HomeWizard Wi-Fi Watermeter
 thing-type.homewizard.watermeter.description = This thing provides the measurement data that is available through the http interface of a HomeWizard Watermeter.
 
 # thing types config
 
+thing-type.config.homewizard.energy_socket.ipAddress.label = Network Address
+thing-type.config.homewizard.energy_socket.ipAddress.description = The IP or host name of the Energysocket.
 thing-type.config.homewizard.energy_socket.ipAddress.label = Network Address
 thing-type.config.homewizard.energy_socket.ipAddress.description = The IP or host name of the Energysocket.
 thing-type.config.homewizard.energy_socket.refreshDelay.label = Refresh Interval
 thing-type.config.homewizard.energy_socket.refreshDelay.description = The refresh interval in seconds for polling the Energysocket.
+thing-type.config.homewizard.energy_socket.refreshDelay.label = Refresh Interval
+thing-type.config.homewizard.energy_socket.refreshDelay.description = The refresh interval in seconds for polling the Energysocket.
 thing-type.config.homewizard.p1_wifi_meter.ipAddress.label = Network Address
 thing-type.config.homewizard.p1_wifi_meter.ipAddress.description = The IP or host name of the P1 Meter.
 thing-type.config.homewizard.p1_wifi_meter.refreshDelay.label = Refresh Interval
 thing-type.config.homewizard.p1_wifi_meter.refreshDelay.description = The refresh interval in seconds for polling the P1 Meter.
 thing-type.config.homewizard.watermeter.ipAddress.label = Network Address
 thing-type.config.homewizard.watermeter.ipAddress.description = The IP or host name of the Watermeter.
+thing-type.config.homewizard.watermeter.ipAddress.label = Network Address
+thing-type.config.homewizard.watermeter.ipAddress.description = The IP or host name of the Watermeter.
+thing-type.config.homewizard.watermeter.refreshDelay.label = Refresh Interval
+thing-type.config.homewizard.watermeter.refreshDelay.description = The refresh interval in seconds for polling the Watermeter.
```

Comment:
Duplicated.

Reply:
Result of duplice thing declaration, fixed.